src/main/entry.js

```diff
@@ -1,4 +1,19 @@
 import { Grid } from '../core/grid.js';
-import { GRID_SIZE } from '../constants.js';
+import { GRID_SIZE, KEY_TO_DIRECTION } from '../constants.js';
 
 const grid = new Grid(GRID_SIZE);
+
+const handleKeyDown = (event) => {
+  const direction = KEY_TO_DIRECTION[event.key];
+  if (!direction) {
```

Comment:
You can do this, but it's up to you.
```suggestion
 if (!direction) return;
``